spec/core/CallTrackerSpec.js

```diff
@@ -143,4 +143,17 @@ describe('CallTracker', function() {
 
     expect(callTracker.mostRecent().args).toEqual(args);
   });
+
+  it('default value for interactionChecked is false', function() {
+    const callTracker = new jasmineUnderTest.CallTracker();
+
+    expect(callTracker.getInteractionChecked()).toBeFalsy();
```

Comment:
Better to use `.toBe(false)` or `.toBeFalse()` here, and `toBe(true)` or `toBeTrue()` in the following spec, since we know exactly what the value should be.